doc/whats_new.rst

```diff
@@ -11,6 +11,8 @@ NEW
 Fixes
 -----
 
+- Region means are no longer zero or casted to Int type
```

Comment:
this is not clear

Reply:
The full sentence is not clear or some specific part?

Reply:
I would write something like

labelsmasker no longer truncates region means to their integral part when input images are of integer type